### PR TITLE
fix: disable text highlight on computer view

### PR DESCRIPTION
**Pull request by kt286**
**before**

image

**after**

image

### CODE REVIEW

**Comment by max-lvs** [comment]
Thanks for your advice, but this definition from dde-file-mananger product design, your advice will be considered. :+1:

File: src/dde-file-manager-lib/views/computerviewitemdelegate.cpp [hunk 1]
```diff
@@ -129,7 +129,7 @@ void ComputerViewItemDelegate::paint(QPainter *painter, const QStyleOptionViewIt
         painter->drawPixmap(option.rect.x() + leftmargin, option.rect.y() + topmargin, icon.pixmap(iconsize));
 
         painter->setFont(par->font());
-        painter->setPen(qApp->palette().color((option.state & QStyle::StateFlag::State_Selected) ? QPalette::ColorRole::HighlightedText : QPalette::ColorRole::Text));
```
Comment (zccrs) [inline src/dde-file-manager-lib/views/computerviewitemdelegate.cpp:131]: The following talk is from the designer:

````
The text color of selected state should be use QPalette::BrightText.
````

Please try it and upload the screenshot for both dark and light theme.

**Review by zccrs — CHANGES_REQUESTED** [review]

File: src/dde-file-manager-lib/views/computerviewitemdelegate.cpp (hunk 1 above)
Comment (kt286) [reply src/dde-file-manager-lib/views/computerviewitemdelegate.cpp:131]: image

image

Comment (zccrs) [reply src/dde-file-manager-lib/views/computerviewitemdelegate.cpp:131]: Yes, it's right.

**Review by Johnson-zs — APPROVED** [review]
Thanks

**Review by zccrs — APPROVED** [review]

**Pull request merged**